lib/Agrammon/Formula/Builtins.pm6

```diff
@@ -25,6 +25,11 @@ sub get-builtins is export {
                     unless $filter-group ~~ Agrammon::Outputs::FilterGroupCollection;
             $filter-group.scale(+$multiplier)
         },
+        add => -> $filter-group, $additor {
+            die "scale operator expects a filter group as its first argument"
```

Comment:
To be consistent the error should be "**add** operator expects..."

Reply:
fixed in 727e39bccff6c2eb9b9b0c9d132db4cd63ca6943